ci/docs.yml

```diff
@@ -0,0 +1,28 @@
+name: uxarray-docs
+channels:
+    - conda-forge
+dependencies:
+    - python=3.9
+    - dask
+    - netcdf4
+    - numba
+    # pinning the numpy version for numba compatibility
+    - numpy>=1.2, <1.22
+    - pathlib
+    - pre_commit
+    - pytest
+    - pytest-cov
+    - scipy
+    - xarray
+    - pip
+    - ipykernel
+    - ipython
+    - jupyter_client
+    - matplotlib-base
+    - sphinx-book-theme
+    - myst-nb
```

Comment:
Could you please document (probably in the PR's description) what this `myst-nb` and some other dependencies (e.g. `sphinx-design`, `nbsphinx`, `sphinxext-opengraph`, etc.) are being used for? That would make a great future reference for whoever needs to understand this infrastructure change.

Reply:
Yeah, can do! I have some of this documented on the geocat team drive, but I can beef up both explanations!